cmd/k8s-operator/sts.go

```diff
@@ -783,6 +785,18 @@ func applyProxyClassToStatefulSet(pc *tsapi.ProxyClass, ss *appsv1.StatefulSet,
 			enableEndpoints(ss, metricsEnabled, debugEnabled)
 		}
 	}
+	if pc.Spec.UseLetsEncryptStagingEndpoint && (stsCfg.proxyType == proxyTypeIngressResource || stsCfg.proxyType == string(tsapi.ProxyGroupTypeIngress)) {
+		logger.Debugf("setting TS_DEBUG_ACME_DIRECTORY_URL to %s", letsEncryptStagingEndpoint)
```

Comment:
nit: Is this log line useful beyond the initial development of the feature? I am fine with leaving it in if you'd like to keep it, just checking that it's intentional

Reply:
That was a debug leftover